kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -4,7 +4,7 @@ local redis = require "resty.redis"
 local policy_cluster = require "kong.plugins.rate-limiting.policies.cluster"
 local reports = require "kong.core.reports"
 local ngx_log = ngx.log
-local shm = ngx.shared.kong_cache
+local shm = ngx.shared.kong_rl_counters
```

Comment:
Other option could be adding a new `config` parameter to rate-limiting plugin where you can configure the `shm` per plugin configuration. Not sure though, should we still supply a default.

Reply:
With `configure_by_lua`, yep! Otherwise, it becomes harder to maintain a custom nginx configuration in sync with the plugins configured in the database...

Reply:
(That or a more powerful templating system that we do not have and tbh, aren't sure of ever wanting at all, as it would make custom nginx templates harder and harder to maintain...)